### ISSUE TITLE
Incorrect x87 lifting

### ISSUE THREAD

**Issue by 0x1F9F1** [Vector35/binaryninja-api#1261]
Binary Ninja Version: 1.1.1486-dev, 747f0dfa
Platform: Windows 10 (10.0)

Some x87 instructions which read memory are being lifted wrong. For example:
```
fld  dword [0x1234] -> x87.push(st0)
fcom dword [0x1234] -> fsub.t{x87com}(st0, st0)
```

**Comment by psifertex** [comment]
Thanks for the title change, I was about to suggest something like that myself. ;-)

**Comment by KyleMiles** [comment]
Fixed in 1.1.1721-dev.
